testing/src/main/java/io/grpc/internal/testing/TestClientStreamTracer.java

```diff
@@ -26,6 +27,7 @@
  */
 public class TestClientStreamTracer extends ClientStreamTracer implements TestStreamTracer {
   private final TestBaseStreamTracer delegate = new TestBaseStreamTracer();
+  private final CountDownLatch outboundHeadersLatch = new CountDownLatch(1);
```

Comment:
consider making this protected.  The rest of this class is designed to be overridable.  

For example the `outboundHeaders()` method below could be overridden,  but have no way to count down this latch.

Reply:
Done